plugins/inputs/ceph/ceph.go

```diff
@@ -79,6 +107,23 @@ func (c *Ceph) SampleConfig() string {
 
 func (c *Ceph) Gather(acc telegraf.Accumulator) error {
 	c.setDefaults()
```

Comment:
I know you didn't write the setDefaults function, but this is actually incorrect and will overwrite any user-provided configuration parameters. Defaults should be set in the init() function. So in init(), you should return the `Ceph` struct with any default options set.

You also should probably be setting GatherAdminSocketStats to true here. If this is the current behavior, then users of the ceph plugin will be broken by the change unless they update their config file (because all bools default to false in Go).